src/shared-components/button/shared-components/container/index.ts

```diff
@@ -35,7 +35,7 @@ const ButtonContainer = styled.div<{ reverse?: boolean; flex?: boolean }>`
     margin-top: 0;
   }
 
-  ${({ reverse }) => reverse && reverseStyles};
+  ${({ reverse = false }) => reverse && reverseStyles};
```

Comment:
Here, for instance, I used ES6 defaults because the Container is a more general component and it seemed appropriate to keep that concern in here instead of components that use it as a subcomponent.